README.md

```diff
@@ -426,6 +424,57 @@ __Example__
 var numbers = flyd.stream(0);
 var squaredNumbers = flyd.map(function(n) { return n*n; }, numbers);
 ```
+### flyd.chain(fn, s)
```

Comment:
What about the FL-standard `s.chain(fn)` or the static  `chain(fn)(s)`?

Reply:
What about them? can you elaborate?

Reply:
Whether I can use them instead?

Reply:
I have never seen the syntax like here in this context. Any motivation for it?

Reply:
There's actually no FL-standard `s.chain(fn)`.

If you read the [spec](https://github.com/fantasyland/fantasy-land#prefixed-method-names), there's a requirement for a prefixed `fantasy-land/chain` to comply with the spec.

`flyd.chain` can be used in a number of ways:

1. directly
```js
flyd.chain(fn, s);
```
2. through pipe
```js
s.pipe(flyd.chain(fn));
```
3. bound to the stream as `fantasy-land/chain`
```js
s['fantasy-land/chain'](fn);
```

All of these ways are functionally equivalent.

Reply:
I mean here:
https://github.com/fantasyland/fantasy-land#chain-method
```js
m.chain(f)
```

As a monad, `flyd.stream` object has to implement the `.chain` method.

I didn't mean the prefixed methods.
They are in addition, more for the library creators and the situation
when you have low level libraries used by higher level ones.
However, there is no higher level library for `flyd`.

I can see the other ways, but none of them is really as usable,
they are more for a low level library.

So I would need to write a micro-wrapper of my own,
which I shall do if the direct methods won't be supported out of the box,
but I've thought it might be useful for more people than just me.

Reply:
the part of the spec I linked says:
> Further in this document unprefixed names are used just to reduce noise.

Meaning it's the prefixed methods that matter. Unprefixed methods are not part of the spec and as such can for the most part be treated as having undefined behaviour.

Now we really have 2 camps when it comes to FP in JS, there's the Haskell camp, and the Scala camp.

The Scala camp prefers method chaining and the Haskell camp prefers function composition.

To me, the pipe method should make both camps happy.

The function composition camp can write:

```js
import {stream} from 'flyd';
import {chain, map, compose, prop} from 'ramda';

const filter$ = stream('');
const result$ = compose(
  map(prop('content')),
  chain(compose(fromPromise, getResult))
)(filter$);
```

and the method chaining camp can write:

```
import {stream} from 'flyd';
import {chain, map, prop} from 'ramda';

const filter$ = stream('');
const result$ = filter$
  .pipe(chain(compose(fromPromise, getResult)))
  .pipe(map(prop('content')));
```

Note: chain and map could also be imported from flyd, but I wanted to show that other libraries supporting the fantasy-land spec can be used too.